taxi/forms.py

```diff
@@ -0,0 +1,45 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+def clean_license(license_number):
+    if len(license_number) == 8:
+        letter = all(letter.isupper() for letter in license_number[:3])

```

Comment:
you can apply `.isupper()` (the same with `isdigit() `)  to the whole string, no need to iterate over each element. as well you need to check if the first three symbols are isalpha()

Reply:
add more descriptive error messages.